src/blocks/hero/deprecated.js

```diff
@@ -6,6 +6,7 @@ import classnames from 'classnames';
 /**
  * Internal dependencies
  */
+import { attributes } from './block.json';
```

Comment:
I think we need to export attributes from index.js

Reply:
Having trouble importing the attributes being exported from `index.js`. Modified the code to export/import just like we're doing in the alert block but the imported attributes continue to be `undefined`. Not sure why yet but since this is only a minor cleanup we can figure this out later.

Leaving it as is so that we can keep moving forward on completing the 1.12.2 milestone.